src/aws/cis-1.2.0/package.json

```diff
@@ -22,7 +22,7 @@
   },
   "devDependencies": {
     "@autocloud/eslint-config": "^0.1.0",
-    "@cloudgraph/sdk": "^0.14.0",
+    "@cloudgraph/sdk": "0.21.0-alpha.3",
```

Comment:
Lets wait to merge this until we have a stable release of the sdk so we dont list an alpha version here